app/models/search/tag_indexer.rb

```diff
@@ -67,7 +68,8 @@ def document(object)
     ).merge(
       has_posted_works: object.has_posted_works?,
       tag_type: object.type,
-      uses: object.taggings_count_cache
+      uses: object.taggings_count_cache,
+      to_wrangle: object.wrangleable? && object.used? && object.unwrangled?
```

Comment:
I'm not sure defining a `to_wrangle` field whose value is computed partially from information we already have indexed (`uses`, `unwrangleable`, `canonical`) is the best way to handle this. 

Why not add an `unwrangled` (or `wrangled` if it's too easy to confuse with `unwrangleable`) field so we build a query using the three fields to get the results we want? That way if, e.g., we only want to include tags with greater than or equal to 3 uses in the mass bins, we'd only need to update the query, not reindex everything to update its `to_wrangle` field.